MovieQuiz/Helpers/UIColor+Extensions.swift

```diff
@@ -1,4 +1,7 @@
 import Foundation
 import UIKit
 
-extension UIColor { }
+extension UIColor {
+    static let ypGreen = UIColor { _ in UIColor(named: "YP Green")! }
+    static let ypRed = UIColor { _ in UIColor(named: "YP Red")! }
```

Comment:
Можно немного короче. Плюс я бы не рекомендовала использовать в названии файлов пробелы. 
```suggestion
    static let ypGreen = UIColor(named: "YP Green")!
    static let ypRed = UIColor(named: "YP Red")!
```